Library/src/Systems/Assets/AssetCollection.cpp

```diff
@@ -241,7 +240,11 @@ void AssetCollectionsResult::FillResultTotalCount(const csp::common::String& Jso
 	{
 		JsonDoc.Parse(JsonContent.c_str());
 
-		if (JsonDoc.HasMember("itemTotalCount"))
+		if (JsonDoc.IsArray())
```

Comment:
Why was this change needed? It has worked up until now without this. Did something change in CHS?

Reply:
It allows us to reuse the same object type `AssetCollectionResult` for both queries that return a single asset collection, and those that return multiple asset collections.

Reply:
I'm not sure I follow. This change looks like 2 different ways of handling JSON that returns multiple results.

Reply:
That's right. This function can now handle two different payloads, one which describes a single asset collection, and one that can return many.